recipes/mapio/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "MapIO" %}
+{% set version = "0.6.1" %}
+{% set sha256 = "c05fe8ae74253decc2fdb0b4129743e119db4594cc6f1bb15fc5b2049f8a8b36" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/usgs/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
The command line options are not needed because MapIO does not use setuptools. Please remove them.